test/e2e-helm/e2e_helm_olm_test.go

```diff
@@ -15,25 +15,88 @@
 package e2e_helm_test
 
 import (
+	"fmt"
+	"os/exec"
+	"path"
 	"path/filepath"
+	"strings"
 
 	. "github.com/onsi/ginkgo" //nolint:golint
+	. "github.com/onsi/gomega" //nolint:golint
 
 	testutils "github.com/operator-framework/operator-sdk/test/internal"
 )
 
-var _ = PDescribe("Integrating Helm Projects with OLM", func() {
+var _ = Describe("Integrating Helm Projects with OLM", func() {
 	Context("with operator-sdk", func() {
+		const operatorVersion = "0.0.1"
+
 		BeforeEach(func() {
-			By("Turning off interactive prompts for all generation tasks.")
+			By("turning off interactive prompts for all generation tasks.")
 			replace := "operator-sdk generate kustomize manifests"
 			testutils.ReplaceInFile(filepath.Join(tc.Dir, "Makefile"), replace, replace+" --interactive=false")
 		})
 
 		AfterEach(func() {
+			By("destroying the deployed package manifests-formatted operator")
+			cleanupPkgManCmd := exec.Command(tc.BinaryName, "cleanup", "packagemanifests",
+				"--operator-version", operatorVersion,
+				"--timeout", "4m")
+			_, _ = tc.Run(cleanupPkgManCmd)
+
+			By("uninstalling CRD's")
+			_ = tc.Make("uninstall")
 		})
 
-		It("Should allow generate the OLM bundle and run it", func() {
+		It("should generate and run a valid OLM bundle and packagemanifests", func() {
+			By("building the bundle")
+			err := tc.Make("bundle")
+			Expect(err).NotTo(HaveOccurred())
+
+			By("validating the bundle")
+			bundleValidateCmd := exec.Command(tc.BinaryName, "bundle", "validate", "./bundle")
```

Comment:
This needs to work universally.
```suggestion
			bundleValidateCmd := exec.Command(tc.BinaryName, "bundle", "validate", "bundle")
```